sky/execution.py

```diff
@@ -437,14 +437,8 @@ def exec(  # pylint: disable=redefined-builtin
     backend_utils.check_cluster_name_not_reserved(cluster_name,
                                                   operation_str='sky.exec')
 
-    status, handle = backend_utils.refresh_cluster_status_handle(cluster_name)
-    with ux_utils.print_exception_no_traceback():
-        if handle is None:
-            raise ValueError(f'Cluster {cluster_name!r} not found.  '
-                             'Use `sky launch` to provision first.')
-        if status != global_user_state.ClusterStatus.UP:
-            raise ValueError(f'Cluster {cluster_name!r} is not up.  '
-                             'Use `sky status` to check the status.')
+    handle = backend_utils.check_cluster_available(cluster_name,
+                                                   'executing task on')
```

Comment:
1. Other operation strings are missing "on"; some need an "of" such as: 
```
» sky queue
...
getting the job queue cluster 'jump3' (status: STOPPED)... skipped.
getting the job queue cluster 'smoke' (status: STOPPED)... skipped.
```
Seems a bit brittle to get the grammar right.

How about
```
Getting the job queue: skipped for cluster 'jump3' (status: STOPPED).
Executing tasks: skipped for cluster 'jump3' (status: STOPPED).
...
```
by passing the following?
```
operation='Getting the job queue'
operation='Executing tasks'
...
```

2. task -> tasks

Reply:
That is a great point! Modified accordingly. Thanks!